include/circt/Dialect/DC/DCCanonicalization.td

```diff
@@ -0,0 +1,20 @@
+//===- DCCanonicalization.td - DC canonicalization ---------*- tablegen -*-===//
```

Comment:
I'll be curious to see what this file is used for.

Reply:
There are some hints in the rationale document about potential canonicalizations.